submissions/iakow/js-dom/main.js

```diff
@@ -0,0 +1,169 @@
+      "The svelte scarlet car retained the ground-breaking semi-automatic gearbox of John Barnard's visually similar 1989 640. Its V12 tended to be outgunned by the V10s of Honda and Renault in 1990, but the chassis was a good one and Alain Prost battled McLaren's Ayrton Senna for the world title.",
+      "That enthralling contest ended when Senna crashed into Prost at the start of the Japanese GP, but it was the closest Ferrari got to ending its title drought until the Jean Todt/Ross Brawn/Michael Schumacher era."
+    ]
+  },
+  {
+    name: "Jordan 191",
+    years: "1991", design: "Gary Anderson",
+    wins: "0",
+    pic: "./img/Jordan_191.jpg",
+    descr: [
+      "Few teams have made such a big impact on their arrival in F1 as Jordan. Designer Gary Anderson's 191 was not only surprisingly rapid, it was beautiful. Shapely curves, which even extended to the distinctive nose and front wing, and the simple 7Up green livery made it an instant modern classic.",
+      "As a new team at a time when there were more F1 cars than spaces on the grid, Jordan initially had to pre-qualify, but the 191 made that a relatively easy task for regular drivers Andrea de Cesaris and Bertrand Gachot. Jordan scored its first points in the Canadian GP and the team ended up fifth in the constructors' table, behind only big hitters McLaren, Williams, Ferrari and Benetton.",
+      "The 191 also helped Michael Schumacher launch his F1 career, with a stunning performance at Spa, and de Cesaris might have won the Belgian GP had it not been for an engine failure, the result of a miscommunication between engine supplier Cosworth and the team."
+    ]
+  },
+  {
+    name: "Lotus 79",
+    years: "1978", design: "Peter Wright, Colin Chapman, Geoff Aldridge, Martin Ogilvie, Tony Rudd",
+    wins: "6",
+    pic: "./img/Lotus_79.jpg",
+    descr: [
+      "Ground-breaking design? Check. Iconic livery? Check. Uncluttered lines? Check. Beautiful engineering? Check.",
+      "The Lotus 79, which refined the ground-effect concept and dominated the 1978 F1 season, ticks all the boxes for this list. It's also even more impressive close up in the flesh than it is in the famous period images of Mario Andretti and Ronnie Peterson.",
+      "Lower, neater and sleeker than the Lotus 78 that preceded it, the 79 won on its debut in the Belgian GP in the hands of Andretti and finished 1-2 next time out in Spain. It was only beaten to pole once during the campaign, though the odd reliability issue and jump-start penalty for Andretti at Monza meant it only won six of the 11 championship rounds it started in 1978.",
+      "By 1979 the opposition had caught up in the ground-effect stakes and the Lotus 79 was limited to four podiums. Even then it looked impressive in its green Martini livery, but it's the JPS-liveried original that deservedly earns our top spot."
+    ]
+  },
+]
+
+const carPhoto = document.querySelector('.photo');
+const carName = document.querySelector('h2');
```

Comment:
You can add some class for this element and make querying by this class because adding a new `h2`element before current element can break your app.